components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -10,6 +10,10 @@ toc = true
     weight = 20
 +++
 
+{{< info >}}
+Chef Automate releases later than 20200326170928 contain the version of IAM described on this page.
```

Comment:
its really this version or later

Reply:
and its prolly me being pedantic, but previous versions had it too.. just folks had to manually upgrade

Reply:
I also think this should be qualified to indicate that previous releases had IAM v2 available via manual upgrade and that from this release (also probably not the one indicated here) forward, users _automatically_ and _exclusively_ had IAM v2.